docs/src/external_mailbox_cmds.md

```diff
@@ -0,0 +1,901 @@
+| chksum     |  u32           |                                         |
+| fips_status|  u32           | FIPS approved or an error               |
+| version    |  u8[32]        | Firmware Version Number in ASCII format |
+
+### MC_DEVICE_CAPABILITIES
+
+Retrieve the device capabilites.
+
+Command Code: `0x4D43_4150` ("MCAP")
+
+*Table: `MC_DEVICE_CAPABILITIES` input arguments*
+| **Name**   | **Type**       | **Description**                         |
+| ---------- | -------------- | --------------------------------------- |
+| chksum     |  u32           |                                         |
+
+*Table: `MC_DEVICE_CAPABILITIES` output arguments*
+| **Name**   | **Type**       | **Description**                         |
+| ---------- | -------------- | --------------------------------------- |
+| chksum     | u32            |                                         |
+| fips_status | u32            | FIPS approved or an error    |
+| caps       | u8[32]         | - Bytes [0:7]: Reserved for Caliptra RT |
+|            |                | - Bytes [8:11]: Reserved for Caliptra FMC |
+|            |                | - Bytes [12:15]: Reserved for Caliptra ROM |
+|            |                | - Bytes [16:23]: Reserved for MCU RT    |
+|            |                | - Bytes [24:27]: Reserved for MCU ROM   |
+|            |                | - Bytes [28:31]: Reserved               |
+
+### MC_DEVICE_ID
+
+Retrieves the device ID.
+
+Command Code: `0x4D44_4944` ("MDID")
+
+*Table: `MC_DEVICE_ID` input arguments*
+| **Name**   | **Type**       | **Description**                         |
+| ---------- | -------------- | --------------------------------------- |
+| chksum     |  u32           |                                         |
+
+*Table: `MC_DEVICE_ID` output arguments*
+| **Name**               | **Type** | **Description**               |
+|------------------------| -------- | ----------------------------- |
+| chksum                 |  u32     |                               |
+| fips_status            | u32      | FIPS approved or an error     |
+| vendor_id              | u16      | Vendor ID; LSB                |
+| device_id              | u16      | Device ID; LSB                |
+| subsystem_vendor_id    | u16      | Subsystem Vendor ID; LSB      |
+| subsystem_id           | u16      | Subsystem ID; LSB             |
+
+### MC_DEVICE_INFO
+
+Retrieves information about the target device.
+
+Command Code: `0x4D44_494E` ("MDIN")
+
+*Table: `MC_DEVICE_INFO` input arguments*
+| **Name**   | **Type** | **Description**                         |
+| ---------- | -------- | --------------------------------------- |
+| chksum     | u32      |                                         |
+| index      | u8       | Information Index:                     |
+|            |          | - `00h` = Unique Chip Identifier       |
+|            |          | Additional indexes are firmware-specific |
```

Comment:
Purely for implementation reasons, it might be good to add a `u8[3]` of padding after, or change to `u32` (and elsewhere).